src/dev-requirements.txt

```diff
@@ -1,3 +1,4 @@
+pylint
 fakeRPiGPIO
 -e git+https://github.com/xtacocorex/CHIP_IO.git#egg=CHIP_IO
-pylint
+pyserial
```

Comment:
This is for CI. For users, it gets installed in setup, if they select proper device.